### PR TITLE
Create vagrantfile

### PR DESCRIPTION
**Pull request by qijinze911**
Create vagrant environment.

### CODE REVIEW

File: Vagrant_Recommendation/.vagrant/machines/default/virtualbox/action_provision [hunk 1]
```diff
@@ -0,0 +1 @@
+1.5:093868e5-1b5a-441a-9926-008f23065eb7
```
Comment (cheryl1223) [inline Vagrant_Recommendation/.vagrant/machines/default/virtualbox/action_provision]: why do we need these files under the `/.vagrant` directory? If it's not needed, please remove them from this PR

File: Vagrant_Recommendation/Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# -*- mode: ruby -*-
```
Comment (cheryl1223) [inline Vagrant_Recommendation/Vagrantfile]: we only need this Vagrantfile, please do not put it under another directory. You can follow https://github.com/nyu-devops/lab-flask-rest to see the architecture of a Flask project.

File: Vagrant_Recommendation/.vagrant/machines/default/virtualbox/action_provision (hunk 1 above)
Comment (cheryl1223) [reply Vagrant_Recommendation/.vagrant/machines/default/virtualbox/action_provision]: You can also update the .gitignore if this directory is always in your local machine

Comment (qijinze911) [reply Vagrant_Recommendation/.vagrant/machines/default/virtualbox/action_provision]: Sorry for add those files, I just try `vagrant up` on my machine and machine create it automatically. And I already update it.

File: Vagrant_Recommendation/Vagrantfile (hunk 2 above)
Comment (qijinze911) [reply Vagrant_Recommendation/Vagrantfile]: I have already update it, thanks for your remind.

**Comment by hi348** [comment]
I am seeing the last two commits are reverted. So can I put this issue back into the backlog?

**Review by cheryl1223 — COMMENTED** [review]
please remove .DS_Store from this commit as I have already set to ignore this file in .gitignore. You can type `git pull` first then to add the Vagrantfile

**Comment by qijinze911** [comment]
I have remove .DS_Store. I do type `git pull` before I add the Vagrantfile. But my fork is open before your change, so that not work for my fork. I  type `git pull upstream master` and now it works. Thank you for reminding.

**Pull request merged**